compose/README.md

```diff
@@ -119,6 +141,12 @@ Access the frontends at
 
 ## Troubleshooting
 
+### Docker
+
+If you are having troubles starting Docker, make sure the user is added
+to `docker` group. See [post-installation steps for Linux](https://docs.docker.com/engine/install/linux-postinstall/) 
+for further reference.
+
```

Comment:
Is this really necessary to document here? This is pretty generic Docker installation stuff

Reply:
I guess not 😄